.github/ISSUE_TEMPLATE/SUPPORT_QUESTION.md

```diff
@@ -1,13 +1,12 @@
 ---
 name: 💬 Support Question
 about: If you have a question, please check out our Slack or StackOverflow!
-
 ---
 
-<!------------^ Click "Preview" for a nicer view! -->
```

Comment:
The `----------^` points to the "Preview" button and was added as a nice touch by one of the committers when we first created these templates.

Reply:
IMHO, that is silly.

Reply:
So should I revert this?